automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppCreationSpec.scala

```diff
@@ -48,6 +47,9 @@ class AppCreationSpec extends GPAllocFixtureSpec with LeonardoTestUtils with GPA
         for {
           _ <- loggerIO.info(s"AppCreationSpec: About to create app ${googleProject.value}/${appName.value}")
 
+          rat <- Ron.authToken()
```

Comment:
🐀

Reply:
Ha maybe I was inspired by [this comment](https://github.com/DataBiosphere/terra-resource-buffer/pull/193#discussion_r744023823)